cmd/action1_test.go

```diff
@@ -1,13 +1,16 @@
-package cmd
+package cmd_test
```

Comment:
I believe this is a common pattern. Our test cannot see internals, so we can be certain we're just testing the api